### ISSUE TITLE
Come to consensus on which hosts.txt to use

### ISSUE THREAD

**Issue by anonimal** [monero-project/kovri#296]
**By submitting this issue, I confirm the following:**

- I have read and understood the [contributor guide](https://github.com/monero-project/kovri/blob/master/doc/CONTRIBUTING.md).
- I have checked that the issue I am reporting can be replicated or that the feature I am suggesting is not present.
- I have checked opened or recently closed [pull requests](https://github.com/monero-project/kovri/pulls) for existing solutions/implementations to my issue/suggestion.

*Place an X inside the bracket to confirm*
- [X] I confirm.

---

Complete chat log [here](https://github.com/monero-project/kovri/files/404774/hosts.txt.log.txt). Naming specification [here](https://geti2p.net/en/docs/naming).

tl;dr
> +zzz depends on your philosophy on content, endorsement, policy, etc

We should come to a consensus on policy.

I personally vote for either *the whole shebang* or for the custom hosts.txt that I will PR. Both have their benefits and drawbacks.

To **start**, we should use the custom list that I will PR. It will contain the essentials to get all Kovri users using IRC, SMTP/POP3, Zerobin, Wiki, and more.

**Comment by EinMByte** [comment]
In terms of the policy (for your custom list), I think it would be wise not to be too liberal. Assuming that users can change the default hosts.txt, of course.

(These are not legal nor political considerations, just a matter of avoiding to scare away users.)

**Comment by olarks** [comment]
For simplicity's sake I think having only "core"  destinations would be wise. If a user wants to continue to explore i2p they are free to add to hosts.txt. Giving the user a wide-range of untested destinations as default should not be warranted.

**Comment by anonimal** [comment]
@olarks also see #337. I should probably close this issue as it's a duplicate.

**Comment by anonimal** [comment]
JFTR: our default subscription is provided by monero. Users will be able to add what they wish after #337 is resolved.
